builder/js/ncf.js

```diff
@@ -336,7 +336,92 @@ $scope.isSelected = function(technique) {
 $scope.isSelectedMethod = function(method) {
   return angular.equals($scope.selectedMethod,method);
 };
+$scope.getSessionStorage = function(){
+  $scope.resetFlags();
+  var matchIndex;
+  var t1,t2 = undefined;
+  t1 = JSON.parse(sessionStorage.getItem('selectedTechnique'));
+  t2 = JSON.parse(sessionStorage.getItem('originalTechnique'));
+  if(t2 !== null && t2.bundle_name === "") t2.bundle_name=undefined;
+  if(t1 !== null){
+    $scope.restoreFlag  = true;
+    $scope.selectedTechnique = angular.copy(t1);
+    $scope.originalTechnique = angular.copy(t2);
+    if($scope.originalTechnique.bundle_name !== undefined){
+    //Not a new technique
+      var deleted = true;
+      for(var i in $scope.techniques){
```

Comment:
then if (existingTechnique !== undefined) {
  $scope.originalTechnique = angular.copy(existingTechnique);
  set saving and isclone to false
  check on existingTechnique and t2 equality